packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -70,12 +70,20 @@ export const CATEGORY_ID_TO_NAME: Record<CategoryID | typeof QUICK_START_CATEGOR
   strategy: 'Strategy'
 }
 
-const RETRO_CATEGORIES: Array<CategoryID> = ['retrospective', 'feedback', 'strategy']
+/**
+ * Defines the list of categories where the 'Create Custom Activity' card is allowed to appear
+ */
+const CREATE_CUSTOM_ACTIVITY_ALLOW_LIST: Array<typeof QUICK_START_CATEGORY_ID | CategoryID> = [
```

Comment:
when we start supporting poker or other custom templates, adding category here will make the `create custom activity` card appear in a given category